### PR TITLE
[ES] Add `Mascots/Gallery`

### PR DESCRIPTION
**Pull request by agatemosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Mascots/Gallery/es.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+- [Mocha in Summer Fanart Contest](https://osu.ppy.sh/community/contests/48)
+
+## Comunidad
+
+Para arte de fans con las mascotas creadas por la comunidad, véase:
+
+- [Fan Art 6: osu!taiko mascot](https://osu.ppy.sh/community/contests/2)
+- [Fan Art 7: Halloween!](https://osu.ppy.sh/community/contests/5)
+- [Fan Art 8: Christmas](https://osu.ppy.sh/community/contests/36)
```
Comment (juankristal) [inline wiki/Mascots/Gallery/es.md:112]: Should be "Navidad"

File: wiki/Mascots/Gallery/es.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+Para arte de fans con mascotas cameo, véase:
+
+- [Mocha in Summer Fanart Contest](https://osu.ppy.sh/community/contests/48)
+
+## Comunidad
+
+Para arte de fans con las mascotas creadas por la comunidad, véase:
+
+- [Fan Art 6: osu!taiko mascot](https://osu.ppy.sh/community/contests/2)
```
Comment (juankristal) [inline wiki/Mascots/Gallery/es.md:110]: "Mascota de osu!taiko" or something along those lines

**Review by juankristal — CHANGES_REQUESTED** [review]
Titles should be translated. Only pointed out a few but there are many still in english.

File: wiki/Mascots/Gallery/es.md (hunk 2 above)
Comment (agatemosu) [reply wiki/Mascots/Gallery/es.md:110]: No lo traduje porque el título del concurso está en inglés aunque tenga la página en español

**Review by juankristal — APPROVED** [review]

**Pull request merged**